tests/integrations/test_dsm.py

```diff
@@ -76,6 +84,7 @@ def assert_checkpoint_presence(hash_, parent_hash, tags):
         logger.info(f"Look for {hash_}, {parent_hash}, {tags}")
 
         for data in interfaces.agent.get_dsm_data():
+            logger.info(data)
```

Comment:
This should probably have a message associated with it? Otherwise it's just an object without explanation